skylines/lib/emails.py

```diff
@@ -0,0 +1,35 @@
+from threading import Thread
+from flask import current_app, copy_current_request_context
+from flask_mail import Message
+
+
+def send_async(msg):
```

Comment:
Sure it sounds good, but I would need to look into that, do you have a suggestion how to do it?